subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -2763,6 +2787,10 @@ SHELL_STATIC_SUBCMD_SET_CREATE(
 		      "AP DPP start auth request as enrollee:\n"
 		      "-p --peer <peer_bootstrap_id>\n",
 		      cmd_wifi_dpp_ap_auth_init, 3, 0),
+	SHELL_CMD_ARG(reconfig, NULL,
```

Comment:
`dpp_reconfig`?

Reply:
Refer to WiFi Alliance Easy Connect cerificate test plan 5.1.19 STAUT Reconfiguration test.
hostap dose not support reconfig using wpa_cli.
So add L2 api to send DPP_RECONFIG iface_ctrl command.

Reply:
In that case why don't we renamed the shell command from `reconfig` to `dpp_reconfig`?

Reply:
because this is in dpp subcommands. The final shell command format will be
$ wifi dpp reconfig
all these commands no need to have dpp_ duplicate prefix.

Reply:
Sorry missed that, thanks.